### ISSUE TITLE
Logit sorting

### ISSUE THREAD

**Issue by fsnelgar** [2018cx/Multi-Level-OT#7]
In the paper the sequence level sorting is described as over all tokens in sequence which appears to be what `improved_sort` [here](https://github.com/2018cx/Multi-Level-OT/blob/d8d298f244395ee8e9550fb446e6b73a9e9a5193/models/distillation_model.py#L17) does.

However in `DistillationLoss` the logits are pre-sorted [here](https://github.com/2018cx/Multi-Level-OT/blob/d8d298f244395ee8e9550fb446e6b73a9e9a5193/models/distillation_model.py#L296) before calling `improved_sort` which means logits are sorted independently for each token, and `improved_sort` doesn't have an effect (you can verify by checking that indices [here](https://github.com/2018cx/Multi-Level-OT/blob/d8d298f244395ee8e9550fb446e6b73a9e9a5193/models/distillation_model.py#L19) are sequential). 

Is this intended or a bug?

Thanks

**Comment by 2018cx** [comment]
The token-level sorting lines were likely uncommented by mistake during code cleanup. This is not intended, and those two lines should be commented out so that sequence-level sorting operates on the original logits. Apologies for the confusion, and thank you for catching it！
